platform/ios/app/MBXState.m

```diff
@@ -0,0 +1,77 @@
+#import "MBXState.h"
+
+NSString *const MBXCamera = @"MBXCamera";
+NSString *const MBXUserTrackingMode = @"MBXUserTrackingMode";
+NSString *const MBXShowsUserLocation = @"MBXShowsUserLocation";
+NSString *const MBXDebugMask = @"MBXDebugMask";
+NSString *const MBXShowsZoomLevelHUD =  @"MBXShowsZoomLevelHUD";
+NSString *const MBXShowsTimeFrameGraph = @"MBXShowsFrameTimeGraph";
+
+
+@implementation MBXState
+
+- (instancetype) init {
+    if ((self = super.self)) {
+        NSMutableDictionary *dictionary = [NSMutableDictionary dictionaryWithObjectsAndKeys: @"", MBXCamera,
```

Comment:
I think initializing with an empty dictionary would be better here; as you have it now you can have 2 types for a key. However...